### ISSUE TITLE
Upgrading to CI 3?

### ISSUE THREAD

**Issue by MBlondin** [ivandiazwm/phpback#67]
Is there any plans on upgrading to CI 3.0.X?

**Comment by ivandiazwm** [comment]
Yes. It's planned to upgrade to CI v3.x in v2.x of PHPBack. It will probably happen in Oct 2016.

**Comment by MBlondin** [comment]
ok.  We were thinking of doing it for our version.  Would you have any pointers or things to be aware of prior to going through it.

**Comment by MBlondin** [comment]
@ivandiazwm I successfully updated PHPBack to CI 3.0.6  I am testing it to make sure it did not break anything major but so far so good.  Would you like to have a copy of it?

**Comment by ivandiazwm** [comment]
Sure! You can send me a copy or a public repo so I can run the selenium tests. Does it has many changes from the application code?

**Comment by MBlondin** [comment]
@ivandiazwm What I'll do is create a branch and push it to that branch

**Comment by azizur** [comment]
@MBlondin Any update on this?
